### PR TITLE
Fix bug in android sample code

### PR DESCRIPTION
**Pull request by jomrazek**
## Motivation
While upskilling I notice this bug
https://issues.jboss.org/browse/AEROGEAR-9055

## What
Fixed sample code

## Why
To provide consumers code without bugs

## How
Fix reference name to correct one

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

**Comment by jomrazek** [comment]
@secondsun @pb82 small bugfix

**Comment by jomrazek** [comment]
@grdryn old https://bower.herokuapp.com/

**Comment by jomrazek** [comment]
@danielpassos can you guys merge this PR? I don't have rights to do it

File: .bowerrc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "registry": "https://registry.bower.io",
```
Comment (pb82) [inline .bowerrc]: Bower is deprecated and they suggest to migrate away from it: https://bower.io/blog/2017/how-to-migrate-away-from-bower/

We could get this PR verified and merged to not rely on it anymore: https://github.com/aerogear/unifiedpush-admin-ui/pull/29/files

Comment (danielpassos) [reply .bowerrc]: @pb82 https://issues.jboss.org/browse/AEROGEAR-8897

Comment (jomrazek) [reply .bowerrc]: This meant to be just workaround to make build pass for checks.

**Pull request merged**